packages/synchro-charts/src/components/sc-dial/sc-dial.tsx

```diff
@@ -0,0 +1,161 @@
+    if (dataStream.data && dataStream.data.length > 0) {
+      const point = dataStream.data[dataStream.data.length - 1];
+      return point;
+    }
+
+    return undefined;
+  };
+
+  getAlarmStream = (dataStream: DataStream): boolean => {
+    return !!this.associatedStreams && this.associatedStreams.some(stream => stream.id === dataStream.id);
+  };
+
+  getBreachedThreshold = (point: DataPoint | undefined, dataStream: DataStream): Threshold | undefined =>
+    breachedThreshold({
+      value: point && point.y,
+      date: isMinimalStaticViewport(this.viewport) ? new Date(this.viewport.end) : new Date(),
+      dataStreams: [this.dataStream],
+      dataStream,
+      thresholds: getThresholds(this.annotations),
+    });
+
+  /**
+   * Emit the current widget configuration
+   */
+  emitUpdatedWidgetConfiguration = (dataStreams?: DataStream[]): void => {
+    const configUpdate: WidgetConfigurationUpdate = {
+      movement: undefined,
+      scale: undefined,
+      layout: undefined,
+      legend: undefined,
+      annotations: this.annotations,
+      axis: undefined,
+      widgetId: this.widgetId,
+      dataStreams: dataStreams ? getDataStreamForEventing(dataStreams) : [this.dataStream],
+    };
+    this.widgetUpdated.emit(configUpdate);
+  };
+
+  /**
+   * On Widget Updated - Persist `DataStreamInfo`
+   *
+   * Emits an event which persists the current `NameValue[]` state into the
+   * data stream info.
+   */
+  onWidgetUpdated() {
+    // Construct the config update with the new names specified.
+    const nameValue = this.names.find(({ id: nameId }) => this.dataStream.id === nameId);
+    const name = nameValue != null ? nameValue.name : this.dataStream.name;
+    const updatedDataStream = { ...this.dataStream, name };
+    this.emitUpdatedWidgetConfiguration([updatedDataStream]);
+  }
+
+  onChangeLabel = ({ streamId, name }: { streamId: string; name: string }): void => {
+    this.names = updateName(this.names, name, streamId);
+    this.onWidgetUpdated();
+  };
+
+  render() {
+    const propertyPoint = this.getPoint(this.dataStream);
+    const alarmStream = this.getAlarmStream(this.dataStream) ? this.dataStream : undefined;
+    const threshold = alarmStream
+      ? getThresholds(this.annotations).filter(a => a.icon)[0] ||
+        this.getBreachedThreshold(propertyPoint, this.dataStream)
+      : undefined;
+    return (
+      <sc-size-provider
+        size={this.size}
+        renderFunc={(size: RectScrollFixed) => {
+          const totalSize = {
+            ...DEFAULT_CHART_CONFIG.size,
+            ...DEFAULT_MARGINS,
+            ...this.size,
+            ...size,
+          };
+          const chartHeight = totalSize.height - THRESHOLD_LEGEND_HEIGHT_PX;
+          const chartSize = {
+            ...totalSize,
+            height: chartHeight,
+          };
+
+          return [
+            <div style={{ height: `${chartSize.height}px`, width: `${chartSize.width}px` }}>
```

Comment:
we can remove the whole render cell pattern - which was previously utilize only because the grid pattern. with no grid, there is no cells

Reply:
Remove cell pattern